ee/vellum_ee/workflows/display/base.py

```diff
@@ -17,6 +17,7 @@
 class WorkflowTriggerType(Enum):
     MANUAL = "MANUAL"
     SLACK_MESSAGE = "SLACK_MESSAGE"
+    COMPOSIO_INTEGRATION_TRIGGER = "COMPOSIO_INTEGRATION_TRIGGER"
```

Comment:
[oos] we should ensure this stayed aligned with https://github.com/vellum-ai/vellum/blob/601c2303e1f2f56a856fc89e49e8056d6438618e/django/app/domain_models/types/workflows.py#L2124